app/domain/errors.rb

```diff
@@ -485,61 +485,61 @@ module AuthnJwt
         code: "CONJ00103E"
       )
 
       FailedToValidateClaimForbiddenClaimName = ::Util::TrackableErrorClass.new(
-        msg: "Failed to validate claim, claim name '{0-claim-name}' is not " \
-             "matching '{1-regex}' regular expression.",
+        msg:  "Failed to validate claim, claim name '{0-claim-name}' is not " \
+              "matching '{1-regex}' regular expression.",
         code: "CONJ00104E"
       )
 
       FailedToValidateClaimClaimNameInDenyList  = ::Util::TrackableErrorClass.new(
-        msg: "Failed to validate claim, claim name '{0-claim-name}' is in denylist '{1-deny-list}'",
+        msg:  "Failed to validate claim, claim name '{0-claim-name}' is in denylist '{1-deny-list}'",
         code: "CONJ00105E"
       )
       FailedToParseMandatoryClaimsMissingInput = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mandatory claims, mandatory claims value is empty, or was not found.",
+        msg:  "Failed to parse mandatory claims, mandatory claims value is empty, or was not found.",
         code: "CONJ00106E"
       )
 
       InvalidMandatoryClaimsFormat = ::Util::TrackableErrorClass.new(
-        msg: "mandatory claims value '{0-mandatory-claims-value}' is in invalid format. " \
-             "The format should be claim names separated with comma.",
+        msg:  "mandatory claims value '{0-mandatory-claims-value}' is in invalid format. " \
+              "The format should be claim names separated with comma.",
         code: "CONJ00107E"
       )
 
       InvalidMandatoryClaimsFormatContainsDuplication = ::Util::TrackableErrorClass.new(
-        msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
+        msg:  "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
         code: "CONJ00108E"
       )
 
       MappingClaimsMissingInput = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, the mapping claims value is empty, or was not found.",
+        msg:  "Failed to parse mapping claims, the mapping claims value is empty, or was not found.",
         code: "CONJ00109E"
       )
 
       MappingClaimsBlankOrEmpty = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, one ore more mapping statements is blank or empty " \
-             "'{0-mapping-claims-value}'.",
+        msg:  "Failed to parse mapping claims, one ore more mapping statements is blank or empty " \
+              "'{0-mapping-claims-value}'.",
         code: "CONJ00110E"
       )
 
       MappingClaimInvalidFormat = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, the mapping claim value '{0-mapping-claim-value}' is in invalid format."\
-             "The format should be 'annotation_name:claim_name'",
+        msg:  "Failed to parse mapping claims, the mapping claim value '{0-mapping-claim-value}' is in invalid format."\
```

Comment:
@sashaCher please provide example of
1. this error with filled in values,
2. what the correct "mapping claims value" should look like

I'm probing in the dark here

Reply:
> "The format should be 'annotation_name:claim_name'"

The error thrown when there's more than one colon in the mapping statement or there's no colon at all